Tests/TestActionsHelper.psm1

```diff
@@ -83,21 +83,29 @@ function YamlTest {
                 $yaml.AppendLine("  $($name):") | Out-Null
                 $yaml.AppendLine("    description: $description") | Out-Null
                 $yaml.AppendLine("    required: $($required.ToString().ToLowerInvariant())") | Out-Null
-                $envLines.AppendLine("        _$($name): `${{ inputs.$($name) }}")
-                if ($type -eq "System.String" -or $type -eq "System.Int32") {
-                    $parameterString += " -$($name) `$ENV:_$($name)"
+                if ($name -eq 'settingsJson') {

```

Comment:
A comment in here would be useful to understand why the special case for `settingsJson`.